server/channels/app/command.go

```diff
@@ -460,6 +460,7 @@ func (a *App) tryExecuteCustomCommand(c request.CTX, args *model.CommandArgs, tr
 	p.Set("text", message)
 
 	p.Set("trigger_id", args.TriggerId)
+	p.Set("root_id", args.RootId)
```

Comment:
Stumbled across this missing parameter as well. Since it was small and simple, fixed it as well.
https://mattermost.atlassian.net/browse/MM-62578